src/EventHub/EventHub/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Vesrion 1.9.2
```

Comment:
Our tool will bump the version. Please just add the changelog under `Upcoming Release`.